app/src/App/utils.tsx

```diff
@@ -167,6 +167,11 @@ export const createEditUserProps = {
   userFormRenderFields: USER_FORM_RENDER_FIELDS,
 };
 
+export const heatlhcareProps = {
+  resourcePageSize: FHIR_RESOURCES_PAGE_SIZE,
+};
+
```

Comment:
we do not need to use a manually defined fhir resources page size, we can set count to total to fetch all records.